src/languages/bash.js

```diff
@@ -7,35 +7,66 @@ Category: common
 */
 
 export default function(hljs) {
-  var VAR = {
+  const VAR = {};
+  const BRACKET_VAR = {
```

Comment:
Looks like `man bash` tends to use "brackets" for `[ ]` and "braces" for `{ }`. So probably better use `BRACE_VAR` here.